rust/crd/src/security.rs

```diff
@@ -39,6 +42,17 @@ pub enum Error {
     SecretVolumeBuild {
         source: stackable_operator::builder::pod::volume::SecretOperatorVolumeSourceBuilderError,
     },
+
+    #[snafu(display("only one authentication method is possible, TLS or Kerberos"))]
```

Comment:
It would be even better top use `SUPPORTED_AUTHENTICATION_CLASS_PROVIDERS` here
```suggestion
    #[snafu(display("only one authentication method is possible at a time, either TLS or Kerberos"))]
```

Reply:
c32b828